### ISSUE TITLE
Node 6 LTS

### ISSUE THREAD

**Issue by sublimator** [destinationstransfers/passkit#7]
AWS lambda is `stuck` on an older version. 

How strict are you about the 8.X requirement? 

Would you accept a pull request to get it working on AWS lambda?

**Comment by tinovyatkin** [comment]
We've created this project to go ahead and use new language features. Node 8 will become LTS in October, so, it's very short period to go back and support Node 6.
Even there is not many features from Node 8 we are actually using in this lib (`util.promisify` and `async/await` as far I can recall) and all of them can be replaced for Node 6, Jests tests and upcoming APN support features are all relies on `async/await` and our targeted server is also Koa which is `>=7.7`. 
So, basically the answer is no, we have no plans to support `Node 6` at this point.
